vpc/tool/identity/ec2.go

```diff
@@ -74,6 +132,8 @@ func (e *ec2Provider) GetIdentity(ctx context.Context) (*vpcapi.InstanceIdentity
 
 	stats.Record(ctx, getIdentityLatency.M(float64(time.Since(start).Nanoseconds())), getIdentitySuccess.M(1))
 
+	go cacheIdentityFiles(ctx, resp, pkcs7)
```

Comment:
I think this should be synchronous, because many titus vpc tool invocations are short lived.

So, I think the flow should go:
1. Call identity cache
2. Cache checks if files are on disk, if files are on disk, return identity from them
3. Fetch identity from IMDS
4. Write out cache
5. Return identity from identity fetched in (3)

Reply:
done.